index.js

```diff
@@ -98,15 +93,27 @@ var Frame = React.createClass({
       setTimeout(this.renderFrameContents, 0);
     }
   },
+  getMountTarget: function() {
+    var doc = this.getDoc();
+
+    if(this.props.mountTarget) {
+      return doc.querySelector(this.props.mountTarget);
+    }
+
+    return doc.body.children[0];
+  },
+  getDoc: function() {
+    return ReactDOM.findDOMNode(this).contentDocument;
```

Comment:
Tried to do this and it resulted in failing tests. I think it's out of scope of this PR, so I reverted that commit. More detailed explanation [here](https://github.com/ryanseddon/react-frame-component/pull/50#issuecomment-239711898)